pkg/config/config.go

```diff
@@ -347,6 +355,160 @@ func (m *Metric) validateMetric(metricIdx int, parent string, discovery *JobLeve
 	return nil
 }
 
+func (c *ScrapeConf) toModelConfig() model.JobsConfig {
```

Comment:
Since YACE has the two entry points there is a risk that this mapping diverges from what's in use in `exporter.go`. IE a field is added to `model.JobsConfig` but not given a config field/mapped from the config. I can't say I know how to easily test for something like this but just something to note.

Reply:
I think I see what you mean. It should be responsibility of config loader to make sure that any field in the config file is mapped correctly/fully to the `model.JobsConfig`. Likewise, when adding or editing fields in `model.JobsConfig` one should go back to the config parser to populate those fields correcly (otherwise the changes might be of no use).